src/Algebra/Graph/AdjacencyIntMap/Algorithm.hs

```diff
@@ -176,10 +188,26 @@ dfsForest = Typed.dfsForest . Typed.fromAdjacencyIntMap
 -- 'forest' (dfsForestFrom [3] $ 'circuit' [1..5] + 'circuit' [5,4..1]) == 'path' [3,2,1,5,4]
 -- @
 dfsForestFrom :: [Int] -> AdjacencyIntMap -> Forest Int
-dfsForestFrom vs = Typed.dfsForestFrom vs . Typed.fromAdjacencyIntMap
+dfsForestFrom vs g = dfsForestFrom' [ v | v <- vs, hasVertex v g ] g
```

Comment:
What happens if we do not filter our non-existent vertices? It's obvious that the implementation will break. If it will, I think a comment would be useful here.

Reply:
I added a remark in the haddocks that seed vertices are ignored if not in the graph. 

If such vertices weren't filtered out, they would be included in the search tree with no children, since `postIntSet` returns an empty set when the vertex isn't in the adjacencymap, I think

Reply:
I actually wanted to say "It's **not** obvious..." in the above comment :-)

Anyway, thanks for clarifying. No need to do anything else.